Sources/Sentry/SentryTransaction.m

```diff
@@ -0,0 +1,130 @@
+#import "SentryTransaction.h"
+#import "NSDate+SentryExtras.h"
+#import "NSDictionary+SentrySanitize.h"
+#import "SentryCurrentDate.h"
+#import "SentryHub.h"
+#import "SentryId.h"
+#import "SentrySpanContext.h"
+#import "SentrySpanId.h"
+#import "SentryTransactionContext.h"
+
+@interface
+
+SentryTransaction ()
+
+/**
+ * This transaction span context
+ */
+@property (nonatomic, strong) SentrySpanContext *spanContext;
```

Comment:
`m`: @marandaneto shouldn't this be a TransactionContext? I know it is different in Java.

Reply:
TransactionContext inherits from SentrySpanContext, the 2 fields, in addition, are used prior to this class, so at runtime, the transaction only needs to know the `SentrySpanContext` fields.
it could be, but there's no reason either.